package.json

```diff
@@ -35,6 +35,7 @@
     "lodash": "4.6.1",
     "mkdirp": "0.5.1",
     "moment": "2.12.0",
+    "path-is-inside": "1.0.1",
```

Comment:
[Not really a one-line though](https://github.com/domenic/path-is-inside/blob/master/lib/path-is-inside.js) :). A custom one-liner is also far from as robust and well-tested (across numerous platforms) as a package too. Unix philosophy + Node ftw :)



> I'm thinking we should only be allowing a-zA-Z0-9_- in usernames.

I thought so as well, but didn't want to make such a decision in this PR.